command/connect/envoy/envoy.go

```diff
@@ -117,6 +118,10 @@ func (c *cmd) init() {
 	c.flags.StringVar(&c.wanAddress, "wan-address", "",
 		"WAN address to advertise in the Mesh Gateway service registration")
 
+	c.flags.Var((*flags.FlagMapValue)(&c.bindAddresses), "bind-address", "Bind "+
```

Comment:
Are these envoy config flags tested anywhere like we do in `agent/config/runtime_test.go`?

Reply:
We test the this command in bootstrap mode but do not have tests for actually execing envoy. Eventually I need to get gateway configurations integrated into our envoy integration tests but that is quite a large undertaking.